contrib/forms/body_composition/new.php

```diff
@@ -135,13 +120,13 @@ function rbinput($name, $value, $desc, $colname)
 <html>
 <head>
 <?php html_header_show();?>
-<link rel=stylesheet href="<?php echo $css_header;?>" type="text/css">
+<link rel=stylesheet href="<?php echo attr($css_header);?>" type="text/css">
 <script language="JavaScript">
 </script>
 </head>
 
 <body <?php echo $top_bg_line;?> topmargin="0" rightmargin="0" leftmargin="2" bottommargin="0" marginwidth="2" marginheight="0">
-<form method="post" action="<?php echo $rootdir ?>/forms/body_composition/new.php?id=<?php echo $formid ?>"
+<form method="post" action="<?php echo att($rootdir) ?>/forms/body_composition/new.php?id=<?php echo attr($formid) ?>"
```

Comment:
Don't html escape full paths (sometimes weird things can happen). Leave $rootdir alone, but the attr() around $formid is correct.